apps/client/src/components/Layouts/BaseLayout.tsx

```diff
@@ -38,12 +39,16 @@ function BaseLayout({ children, checker, heading }: BaseLayoutProps) {
             }}
           >
             <Content>
-              {title && <FormTitle>{title}</FormTitle>}
+              {title && (
+                <FormTitle style={{ maxWidth: "inherit" }}>{title}</FormTitle>
+              )}
               {children}
             </Content>
           </Column>
         </Row>
+      </ContentContainer>
 
+      <HideForPrint>
         <HiddenDebugInfo title="Environment">
```

Comment:
Maybe good to do similar approach for HiddenDebugInfo as HideForPrint sometime in the future. Put it in an atom and rename.

Reply:
I have it noted on the refactor list